src/plugins/data_view_management/public/components/index_pattern_table/index_pattern_table.tsx

```diff
@@ -333,6 +346,43 @@ export const IndexPatternTable = ({
           selection={dataViews.getCanSaveSync() ? selection : undefined}
         />
       </ContextWrapper>
+    </>
+  );
+  if (!hasDataView)
+    displayIndexPatternSection = (
+      <>
+        <EuiSpacer size="xxl" />
+        <NoDataViewsPromptComponent
```

Comment:
🙌

Reply:
Well I was using this since the beginning of the commit :)